### ISSUE TITLE
Doesn't respect chat channels

### ISSUE THREAD

**Issue by anton8604** [the-obsidian/DiscordBridge#28]
Love the plugin, but unfortunately it was a deal breaker that messages from discord to the minecraft server can be seen in every minecraft chat channel. Furthermore, chat within private chat channels on the server are displayed in discord.

If an option can be be added to choose which minecraft chat channel the plugin relays messages to and from, that would be enough to keep DiscordBridge.

We currently use this chat channels plugin:
http://dev.bukkit.org/bukkit-plugins/simplechatchannels/

**Comment by StargateMC** [comment]
Supporting Herochat would also be ideal...

**Comment by anton8604** [comment]
I guess one way that it could be universal is if discord chat could be recognized as player chat, rather than making it a server broadcast

**Comment by jacobwgillespie** [comment]
Yeah, that could help - we'd need a way of mapping Discord users to Minecraft users which I haven't solved yet...

I'll look into SimpleChatChannels and Herochat.  From a brief glance at SCC, it looks like there may not be an API I can hook into, but I'll see what I can do.  Herochat does seem to have an API (at least for detecting what channel a message is coming from).  

If anybody wants to contribute a pull request, I'd be happy to take a look at it, otherwise I'll take a look next time I have some free time.

**Comment by Ionaru** [comment]
Support for TownyChat would be awesome!

http://towny.palmergames.com/towny/townychat/
